src/v/utils/tests/CMakeLists.txt

```diff
@@ -18,9 +19,12 @@ rp_test(
     uuid_test.cc
     vint_test.cc
     waiter_queue_test.cc
+    process_spawn_tests.cc
   LIBRARIES v::seastar_testing_main v::utils v::bytes absl::flat_hash_set
   ARGS "-- -c 1"
   LABELS utils
+  ENV
+    "HANDLE_SIGTERM_SCRIPT=${HANDLE_SIGTERM_SCRIPT}"
```

Comment:
There is `INPUT_FILES` too, which then you can just assume it's in a relative directory to the running test (ie it's at `./handle-sigterm.sh`)

Reply:
I will stick with the ENV var approach for now since it accomplishes the same thing.